tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -46,7 +46,7 @@ Create with anonymous volume
 
 Create with named volume
     ${disk-size}=  Run  docker %{VCH-PARAMS} logs $(docker %{VCH-PARAMS} start $(docker %{VCH-PARAMS} create -v test-named-vol:/testdir ${busybox} /bin/df -Ph) && sleep 10) | grep by-label | awk '{print $2}'
-    Should Be Equal As Strings  ${disk-size}  975.9M
+    Should Contain  ${disk-size}  975.9M
```

Comment:
this would seemingly be unrelated, why are we making all these disk size changes? And why would be we make it less narrow?